### ISSUE TITLE
TypeError: use_args() got an unexpected keyword argument 'locations'

### ISSUE THREAD

**Issue by revmischa** [marshmallow-code/flask-smorest#117]
```
    def decorator(func):
    
        @wraps(func)
        def wrapper(*f_args, **f_kwargs):
            return func(*f_args, **f_kwargs)
    
        # Add parameter to parameters list in doc info in function object
        # The deepcopy avoids modifying the wrapped function doc
        wrapper._apidoc = deepcopy(getattr(wrapper, '_apidoc', {}))
        wrapper._apidoc.setdefault('parameters', []).append(parameters)
    
        # Call use_args (from webargs) to inject params in function
        return self.ARGUMENTS_PARSER.use_args(
>           schema, locations=[location], **kwargs)(wrapper)
E       TypeError: use_args() got an unexpected keyword argument 'locations'

../../../.virtualenvs/jAjXlZQO/lib/python3.7/site-packages/flask_smorest/arguments.py:76: TypeError
```

**Comment by revmischa** [comment]
Downgrading webargs to `5.4.0` fixes it
Yay breaking changes

**Comment by lafrech** [comment]
Indeed.

The fix is easy. See [my comment here](https://github.com/marshmallow-code/webargs/pull/420#issuecomment-553313700).

You're using a beta webargs version. As soon as webargs 6.0.0 is out, I'll release a flask-smorest version including the fix and drop webargs < 6.0.0 support.
